api/handlers/certificates.go

```diff
@@ -97,6 +97,7 @@ func (h AppHandler) CertCreate(c echo.Context) error {
 		return c.JSON(http.StatusInternalServerError,
 			map[string]string{"result": "fail", "message": "The field declared in oidc_claim doesn't exist", "details": err.Error()})
 	}
+	jti, _ := getField(&token, "JTI")
```

Comment:
Hey, @mdjunior! Would it be a good idea to handle error in this case? 🙃

Reply:
In this case, if JTI does not exists, `getField` returns an empty string. I added an error handling to make this more evident.

Reply:
Nice! Thanks for this info! Sounds good to me.